inputs/org.osbuild.oci-archive

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/python3
+"""
+Inputs for oci-archive images
+
+"""
+
+import os
+import pathlib
+import sys
+
+from osbuild import inputs
+
+
+SCHEMA = r"""
+"additionalProperties": false,
+"required": ["type", "origin", "references"],
+"properties": {
+  "type": {
+    "enum": ["org.osbuild.oci-archive"]
+  },
+  "origin": {
+    "description": "The origin of the input (source)",
+    "type": "string",
+    "enum": ["org.osbuild.source", "org.osbuild.source"]
+  },
+  "references": {
+    "description": "Commit identifier",
+    "oneOf": [{
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    }, {
+      "type": "object",
+      "additionalProperties": false,
+      "patternProperties": {
+        ".*": {
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+          }
+        }
+      }
+    }]
+  }
+}
+"""
+
+class OCIInput(inputs.InputService):
+
+    def map(self, store, origin, refs, target, _options):
+        source = store.source("org.osbuild.oci-archive")
+        files = {}
+
+        for ref, data in refs.items():
```

Comment:
`data` is not used. You can either use `keys()` or `for ref, _ in refs.items():`

Reply:
Or even `for ref in refs`